OlinkAnalyze/R/olink_normalization_utils.R

```diff
@@ -1170,67 +1173,138 @@ olink_norm_input_cross_product <- function(lst_df,
 
     # update Olink assay identifiers if cross product normalization ----
 
-    # add combined OlinkID to HT dataset
-    l_ref_name <- names(product_ids)[ref_ids == "ref"]
-    ref_product <- product_ids[ref_ids == "ref"] |> unname()
-    l_ref_oid_rename <- paste0(lst_cols[[l_ref_name]]$olink_id, "_",
-                               ref_product)
+    if (identical(prod_uniq, c("3k", "HT"))
```

Comment:
what about this is unique to 3k and HT, could we generalize this to all products